docs/lib/geo/fragments/js/search.md

```diff
@@ -0,0 +1,209 @@
+<amplify-callout>
+
+**Note:** Amplify Geo is in developer preview and is not intended to be used in production environments. Please reach out to us for any feedback and/or issues [here](https://github.com/aws-amplify/amplify-js/issues)
+
+</amplify-callout>
+
+> Prerequisite: [Install and configure](~/cli/start/install.md) the Amplify CLI
+
+## Provision a Search resource
+
+The primary way to provision Geo resources is through the Amplify CLI. Currently, you need to install the CLI with the `@geo` tag in order to get the Geo functionality. You can use the following command to install this version globally.
+
+```bash
+npm i -g aws-amplify@geo
+```
+
+Once that is complete, you can run the following command from your project's root folder to add a `geo` resource:
+
+```bash
+amplify add geo
+```
+
+```
+? Select which capability you want to add: (Use arrow keys)
+  Map (visualize the geospatial data)
+❯ Location search (search by places, addresses, coordinates)
+```
+
+From here you can follow the prompts to generate your new Place Index to be used for Search.
+
+<!-- TODO: replace with proper link to CLI docs -->
+For more information, you can visit the full [Amplify CLI Geo Search docs](~/lib/geo/search.md).
+
```

Comment:
Same feedback as above, collapse it into the next section as a sentence.

Reply:
Done.